tests/core_tests/chaingen.h

```diff
@@ -466,7 +464,7 @@ inline bool do_replay_events(std::vector<test_event_entry> &events)
 	std::vector<crypto::hash> pool_txs;
 	if(!c.get_pool_transaction_hashes(pool_txs))
 	{
-		MERROR("Failed to flush txpool");
+		std::cout << "Failed to flush txpool" << std::endl;
```

Comment:
I stop here to mention the usage of std:cout. If it must be changed than please change it in the other tests too.